Products/ZenUI3/browser/device_export.py

```diff
@@ -13,10 +13,24 @@
 from Products.ZenUtils.jsonutils import unjson
 from Products.Zuul.facades.devicefacade import DeviceFacade
 from xml.etree.ElementTree import Element, tostring
+from Products.ZenEvents.EventClass import EventClass
 
 log = logging.getLogger('zen.deviceexporter')
 
 
+def event(values):
+    # Get event's types with non-empty count field
+    values = {x:y.get('count') for x,y in values.items() if y.get('count')}
```

Comment:
The `if` clause will be false if `y.get('count')` returns zero.  Is this desired?  If not, you'll need to check explicitly for `None`.